src/App/Utilities/AppHelpers.cs

```diff
@@ -196,6 +197,55 @@ public static async Task<string> SendListOptions(ContentPage page, SendView send
             return selection;
         }
 
+        public static async Task<string> AccountListOptions(ContentPage page, AccountViewCellViewModel accountViewCell)
+        {
+            var vaultTimeoutService = ServiceContainer.Resolve<IVaultTimeoutService>("vaultTimeoutService");
+            var platformUtilsService = ServiceContainer.Resolve<IPlatformUtilsService>("platformUtilsService");
+
+            var userId = accountViewCell.AccountView.UserId;
+
+            List<string> options;
+            if (await vaultTimeoutService.IsLoggedOutByTimeoutAsync(userId) ||
+                await vaultTimeoutService.ShouldLogOutByTimeoutAsync(userId))
+            {
+                options = new List<string> { AppResources.RemoveAccount };
+            }
+            else if (await vaultTimeoutService.IsLockedAsync(userId) ||
+                     await vaultTimeoutService.ShouldLockAsync(userId))
+            {
+                options = new List<string> { AppResources.LogOut };
+            }
+            else
+            {
+                options = new List<string> { AppResources.Lock, AppResources.LogOut };
+            }
+
+            var accountSummary = accountViewCell.AccountView.Email;
+            if (!string.IsNullOrWhiteSpace(accountViewCell.AccountView.Hostname))
+            {
+                accountSummary += "\n" + accountViewCell.AccountView.Hostname;
+            }
+            var selection = await page.DisplayActionSheet(accountSummary, AppResources.Cancel, null, options.ToArray());
```

Comment:
I'd really like if we could remove the dependency on `page` from here to display the action sheet. A service would be ideal to use here that internally searchs for the current page and fires the `DisplayActionSheet`. With this set, we could save of passing the `page` reference through all the overlay and bindings. Dunno if for this PR or for a tech debt task that also refactors other `page` references on other `AppHelpers` methods.

Reply:
Saving this for a tech debt exercise